tests/sentry/utils/email/test_message_builder.py

```diff
@@ -266,6 +266,18 @@ def test_get_built_messages(self):
         results = msg.get_built_messages(["foo@example.com"])
         assert len(results) == 1
 
+    def test_get_built_messages_reply_to(self):
+        msg = MessageBuilder(
+            subject="Test",
+            body="hello world",
+            html_body="<b>hello world</b>",
+            reference=self.activity,
+        )
+        results = msg.get_built_messages(to=["foo@example.com"], reply_to=["abc123@sentry.io"])
+        assert len(results) == 1
+        # Verify Reply-To header is properly set
```

Comment:
this comment also doesn't add anything -- the test name and the assertion below it are fairly straightforward on what is being tested

generally comments should say _why_ not _what_